plugin-redfish/rfphandler/subscribeEvents.go

```diff
@@ -102,31 +103,31 @@ func deleteMatchingSubscriptions(device *rfputilities.RedfishDevice) {
 	device.Location = "https://" + device.Host + "/redfish/v1/EventService/Subscriptions"
 	redfishClient, err := rfputilities.GetRedfishClient()
 	if err != nil {
-		log.Println("error: internal processing error:", err)
+		log.Error("Internal processing error:" + err.Error())
 		return
 	}
 
 	//Get Subscription details to check if it is really ours
 	resp, err := redfishClient.GetSubscriptionDetail(device)
 	if err != nil {
-		log.Println(err.Error())
+		log.Error(err.Error())
 		return
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
-		errorMessage := fmt.Sprintf("error: while getting subscription details for URI: %v got %v", device.Location, resp.StatusCode)
-		log.Println(errorMessage)
+		errorMessage := fmt.Sprintf("Unable to get subscription details for URI: " + device.Location + " got " + strconv.Itoa(resp.StatusCode))
```

Comment:
fmt.Sprintf converts and concatenates the passed arguments into a string. So we can avoid the use of 'strconv.Itoa'

Reply:
I dont like that but changed it